test/tla/cli-integration-tests.md

```diff
@@ -1873,6 +1873,22 @@ $ apalache-mc check --length=0 --inv=AllTests TestSets.tla | sed 's/[IEW]@.*//'
 EXITCODE: OK
 ```
 
+### check TestCommunityFunctions.tla reports no error (array-encoding)
+
+```sh
+$ apalache-mc check --length=0 --inv=AllTests TestCommunityFunctions.tla | sed 's/[IEW]@.*//'
+...
+EXITCODE: OK
+```
+
+### check TestFiniteSetsExt.tla reports no error (array-encoding)
+
+```sh
+$ apalache-mc check --length=0 --inv=AllTests TestFiniteSetsExt.tla | sed 's/[IEW]@.*//'
```

Comment:
Maybe I'm ignorant, but where do these choose the array encoding?

Reply:
This is how the integration tests are running now. If `(array-encoding)` is present, then the both encodings are used; otherwise, only the oopsla19 encoding is used. @shonfeder, is it a correct description?

Reply:
Correct! We run one job for each of the encodings on Ubuntu. In the job runs for the array encoding, only the tests with that string in the name are run.

Reply:
Aaaah, good to know!